### PR TITLE
Fixes the description about `id` key in Issue Forms.

### LINKED ISSUE #9004: Syntax for GitHub's form schema is incorrect with relation to IDs
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's Code of Conduct.

### What article on docs.github.com is affected?

https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema

### What part(s) of the article would you like to see updated?

When trialling out the `.yaml` issue form templates, I came across the following error:
image

As you can see, it states that: `body[0]: id is not a permitted attribute.`

Yet here: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema#attributes-4, `id` is explicitly an allowed attribute. If I move the ID to the `type/attributes` level of the `checkboxes` entry I get:

image

The same also happens on `dropdown` types:

image

I would assume the same is for all types that have `id` listed as an attribute.

### Additional information

I'd be happy to raise a PR to change the docs, but I'm not sure if the docs are right and the implementation is wrong, or the implementation is right and the docs are wrong 😄

### PR DESCRIPTION
**Pull request by thispsj**
### Why:

Closes #9004 

### What's being changed:

The `id` key is used in issue forms with every type (excluding markdown). But the docs mention that the `attributes` key should contain it which is wrong. The `id` key is a top-level key in each type.

### Check off the following:

- [x] I have reviewed my changes in staging (look for the latest deployment event in your pull request's timeline, then click **View deployment**).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### CODE REVIEW

**Review by ciaransweet — DISMISSED** [review]
Nice one @thispsj !

**Comment by ramyaparimi** [comment]
@thispsj Thanks so much for opening a PR! I got this triaged for review :zap:

**Review by quasialaw — DISMISSED** [review]

**Review by ghost — DISMISSED** [review]

**Comment by danaikhamkao721** [comment]
0@

**Comment by ramyaparimi** [comment]
@thispsj Thanks so much for opening a PR! This was triaged for review :zap:

**Comment by ramyaparimi** [comment]
@thispsj Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Comment by ampon0001** [comment]
ampon sangseng

**Review by ampon0001 — COMMENTED** [review]
ampon0001

File: content/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema.md
Comment (stevecat) [inline content/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema.md:64]:
```suggestion
| `id` | The identifier for the element, except when `type` is set to `markdown`. {% data reusables.form-schema.id-must-be-unique %} If provided, the `id` is the canonical identifier for the field in URL query parameter prefills. | Optional | String | {% octicon "dash" aria-label="The dash icon" %} | {% octicon "dash" aria-label="The dash icon" %} |
```

**Review by guntrip — DISMISSED** [review]
This is fantastic, thank you @thispsj! ✨  

I've made one little tweak which I'll apply in a second before merging and getting these changes deployed.

Thanks again for taking the time to contribute to our documentation!

File: content/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema.md
Comment (stevecat) [inline content/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema.md:64]:
```suggestion
| `id` | The identifier for the element, except when `type` is set to `markdown`. {% data reusables.form-schema.id-must-be-unique %} If provided, the `id` is the canonical identifier for the field in URL query parameter prefills. | Optional | String | {% octicon "dash" aria-label="The dash icon" %} | {% octicon "dash" aria-label="The dash icon" %} |
```

Comment (stevecat) [inline content/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema.md:64]:
```suggestion
| `id` | The identifier for the element, except when `type` is set to `markdown`. {% data reusables.form-schema.id-must-be-unique %} If provided, the `id` is the canonical identifier for the field in URL query parameter prefills. | Optional | String | {% octicon "dash" aria-label="The dash icon" %} | {% octicon "dash" aria-label="The dash icon" %} |
```

**Comment by stevecat** [comment]
@thispsj :wave: Would you mind enabling the checkbox to allow maintainer edits? Then I'll be able to commit the suggested change above and get this merged in. Thank you!

https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork

**Comment by thispsj** [comment]
@stevecat done.

**Comment by thispsj** [comment]
@stevecat you can enable automerge now I guess

**Review by guntrip — APPROVED** [review]
@thispsj Perfect, thanks so much for making those changes! I'll get this merged now ✨

**Pull request merged**
